app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,20 @@
 module Api
   class AuthenticationsController < BaseController
+    def create_resource(_type, _id, data)
+      validate_auth_attrs(data)
+      klass = data['type'].constantize
+      # TODO: Temporary validation - remove
+      raise 'type not currently supported' unless klass.respond_to?(:create_in_provider_queue)
+      klass.create_in_provider_queue(parse_id(data['manager'], :providers), data.except('type', 'manager'))
```

Comment:
is authentications always targeted to manager type CI's or could they belong to other types ?  if so, I wonder if we should have a more generic reference (maybe "resource" : { "href" : ... })